src/couch_jobs/src/couch_jobs.erl

```diff
@@ -104,6 +105,12 @@ get_job_state(Tx, Type, JobId) when is_binary(JobId) ->
     end).
 
 
+get_active_jobs(Tx, Type) ->
+    couch_jobs_fdb:tx(couch_jobs_fdb:get_jtx(Tx), fun(JTx) ->
+        couch_jobs_fdb:get_active_since(JTx, Type, {versionstamp, 0, 0})
```

Comment:
If this returns extra info besides JobIds, let's strip it out so it's just a list of JobIds. So it would basically be `[JobId1, JobId2, ...]`. It might make the caller's logic simpler so they don't have to unpack stuff they don't care about.